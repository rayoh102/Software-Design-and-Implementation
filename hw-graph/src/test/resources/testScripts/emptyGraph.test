# Tests a graph with no nodes

# Create the graph
CreateGraph graph1
