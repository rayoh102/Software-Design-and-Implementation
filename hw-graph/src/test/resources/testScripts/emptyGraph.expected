# Tests a graph with no nodes

# Create the graph
created graph graph1